nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -556,37 +556,6 @@ define([
             return bus;
         }
 
-        /**
-         * Gets the collection of file path parameter validation options. This is returned as
-         * an Array of objects, one for each file path row. Each object has a key for each
-         * file path id, and value is the options for that input.
-         *
-         * Currently this only sets the invalidValues option for text validation. I.e., the
-         * files that are not available.
-         * @returns a list of file path options for each file input row.
-         */
-        function getFilePathOptionsForValidation() {
```

Comment:
Isn't there another function in `configure.js` that's very similar to something in `appSpecUtil.js`? It may not be similar enough to get rid of the function... I don't remember the details.

Reply:
I'm pretty sure I removed it? Or at least I meant to.

Reply:
Yeah, not far below this comment. :D